src/index.js

```diff
@@ -1,36 +1,132 @@
 'use strict';
 
-const $tagsInput = document.getElementById('tags-input');
-const $tagsList = document.getElementById('tags-list');
-const tagsArr = ['1', 'super tag'];
+const $app = document.getElementById('app');
+const $tagsList = document.getElementById('initial-list');
+const defaultTags = ['tag1','some awesome tag'];
+var simpleInputTags = [];
+var inputTagsWithDefaul = defaultTags.slice();
 
-$tagsInput.addEventListener('keyup', ({key, target}) => {
-  if (key === 'Enter' && target.value.trim()) {
-    const $el = createTagElement(target.value);
-    const isElementExist = tagsArr.includes(target.value);
+$app.addEventListener('click', ({target}) => {
+  const isRemoveBtn = target.classList.contains('remove-btn');
 
-    if (!isElementExist) {
-      $tagsList.appendChild($el);
-      tagsArr.push(target.value);
-      target.value = '';
-    }
+  if (isRemoveBtn) {
+    const $foo = target.parentElement.parentElement;
+    const $tag = target.parentElement;
+      if (confirm('Are you sure want to delete this super tag?')) {
```

Comment:
in most cases try to avoid using blocking popup like "alert", "confirm", etc
because of it brings bad user experience to the project